model/generate.go

```diff
@@ -281,6 +281,9 @@ func (g *GeneratedProject) saveNewBuildsAndTasks(ctx context.Context, v *Version
 		ActivationInfo: activationInfo,
 		SyncAtEndOpts:  syncAtEndOpts,
 		GeneratedBy:    g.Task.Id,
+		// If the parent generator is required to finish, then its generated
+		// tasks inherit that requirement.
+		AllTasksAreEssentialToComplete: g.Task.IsEssentialToFinish,
```

Comment:
Same for this field - can we just inspect the requester directly?

Reply:
Same reason as above.